commitlint.config.js

```diff
@@ -1 +1,7 @@
-module.exports = { extends: ['@commitlint/config-conventional'] };
+module.exports = {
+  extends: ['@commitlint/config-conventional'],
+  rules: {
+    'body-max-line-length': [2, 'always', 200],
+    'footer-max-line-length': [2, 'always', 200],
```

Comment:
Can we change this to `1` for warning?
https://commitlint.js.org/#/reference-rules

Reply:
Sure.